### PR TITLE
06 get api articles article id comments

### PR DESCRIPTION
**Pull request by SoranRaof**

### CODE REVIEW

File: __tests__/api.test.js [hunk 1]
```diff
@@ -22,20 +24,167 @@ describe('GET /api/topics', () => {
...
+})
 
+describe('GET /api/articles/:article_id', () => {
+    test('responds with a 200 status code', () => {
+        return request(app)
+        .get('/api/articles/1')
+        .expect(200)
+    })
+    test('get /api/articles/:article_id request responds with an array of comments objects with the correct properties', () => {
+        return request(app)
```
Comment (HannahBurks) [inline __tests__/api.test.js:109]: I would also add a test for if given a valid ID but had no comments that it returns an empty comments array (200)

File: __tests__/api.test.js [hunk 2]
```diff
@@ -22,20 +24,167 @@ describe('GET /api/topics', () => {
...
+    })
+})
 
+describe('GET /api/articles/:article_id', () => {
+    test('responds with a 200 status code', () => {
+        return request(app)
+        .get('/api/articles/1')
+        .expect(200)
+    })
+    test('get /api/articles/:article_id request responds with an array of comments objects with the correct properties', () => {
```
Comment (HannahBurks) [inline __tests__/api.test.js:108]: Also add tests for
Status 400, invalid ID, e.g. string of "not-an-id"
Status 404, non existent ID, e.g. 0 or 9999

**Review by HannahBurks — CHANGES_REQUESTED** [review]
Hey Soran, nice work, just some extra tests I would like you to add. 😄
